composer.json

```diff
@@ -18,15 +18,15 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "illuminate/http": "^6.0",
-        "illuminate/routing": "^6.0",
-        "illuminate/session": "^6.0",
-        "illuminate/support": "^6.0",
-        "illuminate/view": "^6.0"
+        "php": ">=7.2.5",
+        "illuminate/http": "^6.0|^7.0",
```

Comment:
should be double pipe https://github.com/composer/composer/issues/6755

https://getcomposer.org/doc/articles/versions.md#version-range

Reply:
Both syntax are supported. And while outdated, Laravel and the community mostly uses the single pipe.